server/services/alert_service/aws_sns/text_messaging.go

```diff
@@ -0,0 +1,52 @@
+package awssns
+
+import (
+	"main/server/request"
+	"main/server/response"
+	"main/server/utils"
+	"os"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/sns"
+	"github.com/gin-gonic/gin"
+)
+
+func SendSMS(ctx *gin.Context, reqBody request.AwsTextMessagingRequest) {
+
+	// Create Session and assign AccessKeyID and SecretAccessKey
+	sess, err := session.NewSession(&aws.Config{
```

Comment:
Will we create a new session each time this method is called?

Reply:
Resolved sir. Created another function which creates a session and assigned it to global variable .